client/assets/javascripts/__tests__/modules_new/ButtonMenu.test.js

```diff
@@ -0,0 +1,40 @@
+import { describe, expect, it, jest } from '@jest/globals'
+import MOJButtonMenu from '../../modules_new/ButtonMenu'
+
+describe('MOJButtonMenu', () => {
+  describe('init', () => {
+    it('should call ButtonMenu if there are more than 2 children', () => {
+      document.body.innerHTML = `
+        <div class="moj-button-menu">
+          <div class="moj-button-menu__wrapper">
+            <a href="#" role="button" class="govuk-button">
```

Comment:
Worth avoiding  `<a href="#" role="button"` and makign it an actual `<button>`

Reply:
Seems like some of the `<a>` tags with the `role=button` have `href=""` values like here:

```
// client/templates/Admin/Client/Client/archived.html.twig:42
<a href="{{ path('admin_client_unarchived', {id: client.id}) }}" role="button" data-module="govuk-button" class="govuk-button govuk-button--secondary">
    Un-archive client
</a>
```

Reply:
Yeah, some weird mixed messages in that code. If it's a link, then it goes somewhere, so it doesn't need the `role="button"` even if it is styled to look like one.

Reply:
Looking at that code it may also mean we may be doing state changes (to unarchive) on a GET, which is bad.